lib/discovery.js

```diff
@@ -211,25 +216,23 @@ function mixinDiscovery(DB2, db2) {
   function queryColumns(schema, table) {
     var sql = null;
     if (schema) {
-      sql = paginateSQL('SELECT tabschema AS "owner",' +
-        ' tabname AS "tableName",' +
-        ' colname AS "columnName",' +
+      sql = paginateSQL('SELECT TRIM(tabschema) AS "owner",' +
+        ' TRIM(tabname) AS "tableName",' +
+        ' TRIM(colname) AS "columnName",' +
         ' typename AS "dataType",' +
         ' length AS "dataLength",' +
-        // ' numeric_precision AS "dataPrecision",' +
```

Comment:
Is this deletion intentional? If so, should be in a seperate commit since it's unrelated to trim changes.